src/OpenTok/OpenTok.php

```diff
@@ -520,6 +521,25 @@ public function getStream($sessionId, $streamId)
         
     }
 
+    /**
+     * Returns a StreamList Object for the given session ID.
+     * 
+     * @param String $sessionId The session ID.
+     *
+     * @return StreamList A StreamList object. Call the items() method of the StreamList object
+     * to return an array of Stream objects.     
+     */
+
+    public function listStreams($sessionId)
+    {
+        Validators::validateSessionIdBelongsToKey($sessionId, $this->apiKey);
```

Comment:
why validate with api key?

Reply:
Validating for because the jwt must be signed with the right `apiKey` and `secret`

Reply:
we don't do this everywhere and is not consistent with REST codes.
approving with that reservation